openid-4-verifiable-credential-issuance-1_0.md

```diff
@@ -175,95 +175,103 @@ The following subsections illustrate some of the authorization flows supported b
 
 ## Authorization Code Flow {#authorization-code-flow}
 
-Below is a diagram of a Credential issuance using the Authorization Code Flow, using grant type `authorization_code` as defined in [@!RFC6749].
+The Authorization Code Flow uses the grant type `authorization_code` as defined in [@!RFC6749] to issue Access Tokens.
 
-The diagram shows how a Wallet-initiated flow use case as described in (#use-case-1) is implemented with the Credential Issuance API defined in this specification. Note that the diagram does not illustrate all the optional features of this specification.
+Figure 1 shows the two flows that can be implemented for the issuance of the Credentials, as outlined in this specification:
```

Comment:
I'm confused by the attempt to integrate two flows into one section, the original intend of these section was to illustrate one scenario each. Perhaps we can frame it a bit differently, in the sense of it is one flow with two different start options.